### PR TITLE
add the ability to force the traffic from/to the server to be json, for better sniffeability

### PR DESCRIPTION
**Pull request by dothebart**
This can be observed while sniffing the tests, i.e. like this:
```
./scripts/unittest shell_client --forceJson true   --test tests/js/client/shell/shell-async-request.js --sniff sudo  --sleepBeforeStart 5
```

### CODE REVIEW

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/6713/

File: arangosh/Shell/ClientFeature.cpp [hunk 1]
```diff
@@ -104,6 +105,10 @@ void ClientFeature::collectOptions(std::shared_ptr<ProgramOptions> options) {
                      "for a password",
                      new StringParameter(&_password));
 
+  options->addOption("--server.force-json",
+                     "force not to use velocypack for better debugability",
+                     new BooleanParameter(&_forceJson));
```
Comment (jsteemann) [inline arangosh/Shell/ClientFeature.cpp]: let's make this a hidden option.
also add `.setIntroducedIn(30600);`.

File: arangosh/Shell/V8ClientConnection.h [hunk 2]
```diff
@@ -162,6 +163,7 @@ class V8ClientConnection {
   fuerte::ConnectionBuilder _builder;
   std::shared_ptr<fuerte::Connection> _connection;
   velocypack::Options _vpackOptions;
+  bool _forceJson;
```
Comment (jsteemann) [inline arangosh/Shell/V8ClientConnection.h]: this variable should be initialized to some value

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/6729/

File: arangosh/Shell/ClientFeature.cpp [hunk 3]
```diff
@@ -104,6 +105,12 @@ void ClientFeature::collectOptions(std::shared_ptr<ProgramOptions> options) {
                      "for a password",
                      new StringParameter(&_password));
 
+  options->addOption("--server.force-json",
+                     "force not to use velocypack for better debugability",
```
Comment (Simran-B) [inline arangosh/Shell/ClientFeature.cpp]:
```suggestion
                     "force to not use VelocyPack for easier debugging",
```

**Comment by dothebart** [comment]
testrun green except for unrelated problems.

**Pull request merged**